### DISCUSSION TITLE
Integration of Winget with the GitHub Packages registry.

### DISCUSSION THREAD

**Discussion by niStee** [microsoft/winget-cli#4162] [Ideas] [answered]
Efficiency and Automation: By integrating Winget with the GitHub Packages registry, you can automate the process of updating your software on users’ machines. Whenever you push a new release to your GitHub Packages, Winget can automatically fetch the latest version and make it available for users to install. This eliminates the need for manual updates and ensures that your users always have access to the latest version of your software.

Ease of Use: Winget is a popular package manager among Windows users due to its simplicity and ease of use. By making your software available through Winget, you can provide a convenient and familiar installation process for your users.

Visibility and Reach: GitHub is one of the largest and most active developer communities in the world. By hosting your packages on GitHub, you can increase the visibility of your software and reach a wider audience.

Use Case: Consider a scenario where you’re maintaining a popular open-source tool. You frequently release updates and new features, and you want to ensure that your users can easily update to the latest version. By integrating Winget with your GitHub Packages registry, users can simply run a command in their terminal to fetch and install the latest release.

**Comment by Trenly** [comment, marked as answer]
You can integrate `wingetcreate` as part of a GitHub workflow. Here is how [PowerToys](https://github.com/microsoft/PowerToys/blob/main/.github/workflows/package-submissions.yml) automated submissions.

**Reply by niStee** [reply]
Consider adding this information to the documentation located at
https://docs.github.com/en/actions/publishing-packages/.
